cmake/setup_custom_targets.cmake

```diff
@@ -99,6 +99,16 @@ IF(NOT DEAL_II_COMPONENT_PACKAGE)
     )
 ENDIF()
 
+#
+# Provide an indentation target for indenting uncommitted changes and changes on
+# the current feature branch
+#
+ADD_CUSTOM_TARGET(indent-branch
+  WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+  COMMAND ./contrib/utilities/indent-branch
+  COMMENT "Indenting recently changed files in the deal.II directories"
```

Comment:
Would it be worth listing this also for `make help` in line 148 of the current file?

Reply:
Yes.